docs/syntax_and_semantics/platform_support.md

```diff
@@ -14,41 +14,56 @@ Tier 1 platforms can be thought of as “guaranteed to work”. Specifically the
 * Automated testing is set up to run tests for the platform.
 * Documentation for how to use and how to build the platform is available.
 
-| Target | Compiler | Std | Description |
-| ------ | -------- | --- | ----------- |
-| x86_64-darwin | ✓ | ✓ | 64-bit OSX (10.7+, Lion+) |
-| x86_64-linux-gnu | ✓ | ✓ | 64-bit Linux (kernel 2.6.18+, GNU libc) |
+Only maintained operating system versions are fully supported. Obsolete versions are not guaranteed to work
+and drop into *Tier 2*.
+
+| Target | Description | Supported versions |
+| ------ | ----------- | ------------------ |
+| `x86_64-darwin` | x64 macOS (Intel) | 11+<br> *(testing only on 11; expected to work on 10.7+)* |
```

Comment:
I think it's relevant to mention which version we're actually testing on. We're currently not testing macOS 12 (crystal-lang/crystal#12570), so liability of this OS version is similar as for `aarch64-darwin` which is *Tier 2*.

Sure, we can add a *Comment* column here as well with the individual aspects. I just thought we can skip that because for *Tier 1* everything needs to be ticked. Maybe it's more consistent that way...